[2022-07-04 15:53:49,649] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_table scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-04 15:53:49,659] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_table scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-04 15:53:49,659] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 15:53:49,659] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-04 15:53:49,660] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 15:53:49,680] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-03 00:00:00+00:00
[2022-07-04 15:53:49,685] {standard_task_runner.py:52} INFO - Started process 417 to run task
[2022-07-04 15:53:49,696] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'create_table', 'scheduled__2022-07-03T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpm487pqx2', '--error-file', '/tmp/tmpgepihj94']
[2022-07-04 15:53:49,697] {standard_task_runner.py:80} INFO - Job 21: Subtask create_table
[2022-07-04 15:53:49,808] {task_command.py:370} INFO - Running <TaskInstance: first_dag.create_table scheduled__2022-07-03T00:00:00+00:00 [running]> on host b631230cc7b5
[2022-07-04 15:53:49,905] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-03T00:00:00+00:00
[2022-07-04 15:53:49,916] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-04 15:53:49,919] {dbapi.py:213} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );, parameters: None
[2022-07-04 15:53:49,921] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-04 15:53:49,936] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=create_table, execution_date=20220703T000000, start_date=20220704T155349, end_date=20220704T155349
[2022-07-04 15:53:49,993] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-04 15:53:50,025] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
